### PR TITLE
Update AccessKit to 0.11 from 0.9.

### PR DESCRIPTION
**Pull request by waywardmonkeys**
This paves the way for updating it within Xilem as well (once that also updates to a current Glazier).

### CODE REVIEW

**Comment by waywardmonkeys** [comment]
Normally, this would have required a semver bump, but since this crate isn't published, it seems like that shouldn't be needed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,3 +4,4 @@
 
 - Add wgpu triangle example ([#53] by [@lord])
 - Add thread-safe app handle ([#90] by [@clavin])
+- Updated AccessKit to 0.11 ([#108] by [@waywardmonkeys])
```
Comment (jneem) [inline CHANGELOG.md:7]: Unfortunately, these links do not get added automatically; could you add them below? (The previous two  changelog entries are broken, so don't follow their example)

**Review by jneem — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (waywardmonkeys) [reply CHANGELOG.md:7]: Done. Once this lands, I don't mind submitting a second PR to fix the links for the others.

**Review by jneem — APPROVED** [review]
Thanks!

**Pull request merged**